opm/output/Cells.hpp

```diff
@@ -32,6 +32,10 @@ class SimulationDataContainer;
 namespace data {
 
     struct CellData {
+        CellData(std::string name, UnitSystem::measure dim, std::vector<double> data)
```

Comment:
In theory, I believe you are correct, but I got a compilation error using my gcc-5.4.0 without this constructor.